Makefile

```diff
@@ -18,6 +18,7 @@ purge:
 
 format:
 	mix format
+	mix format priv/repo/seeds.exs
```

Comment:
Did you try changing `.formatter.exs` with 

```
inputs: ["mix.exs", "config/*.exs", "priv/*/seeds.exs"]
```

Reply:
oh, that does it! I thought these were configuration files for the formatter, not the actual files to format. I figured those should go in `subdirectories` 🤔